W-16515585 RPA Manager release notes 2024-08/1. (#407)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

> [[august-15-2024]]
> == August 15, 2024

Reviewer: ```suggestion
[[august-19-2024]]
== August 19, 2024
```

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * RPA Manager now offers additional types of process trigger executions like interval trigger and cron trigger. These are supported by bots running version 1.7.0 or later.

Reviewer: ```suggestion
* You can now configure additional types of process executions: interval trigger, cron trigger, and on-demand executions. 
+
These are supported by bots running version 1.7.0 or later.
```
